tests/rptest/tests/franz_go_verifiable_test.py

```diff
@@ -172,7 +164,7 @@ def _workload(self, segment_size):
         assert self._seq_consumer.consumer_status.valid_reads >= wrote_at_least
         assert self._rand_consumer.consumer_status.total_reads == self.RANDOM_READ_COUNT * self.RANDOM_READ_PARALLEL
 
-    @cluster(num_nodes=4, log_allow_list=KGO_LOG_ALLOW_LIST)
+    @cluster(num_nodes=4, log_allow_list=RESTART_LOG_ALLOW_LIST)
     @matrix(segment_size=[100 * 2**20, 20 * 2**20])
     def test_si_without_timeboxed(self, segment_size: int):
```

Comment:
This test does not need `restart log allow list` because the nodes do not restart.

Reply:
makes sense 👍